DEVELOPMENT.md

```diff
@@ -30,61 +43,60 @@ flowchart LR
 ```
 
 During test run, the test suites:
```

Comment:
(Comment about the existing document) May be worth being more specific? Or more specific about the abstract description:

```suggestion
During a test run, each integration test generally performs these steps:
```